tests/containers/containers_test.go

```diff
@@ -59,7 +59,7 @@ func TestPulumiDockerImage(t *testing.T) {
 			}()
 
 			stackName := fmt.Sprintf("%s/container-%s-%x", stackOwner, template, time.Now().UnixNano())
-			e.RunCommand("pulumi", "new", template, "-f", "-s", stackName)
+			e.RunCommand("pulumi", "new", template, "yes", "-f", "-s", stackName)
```

Comment:
This needs to be `--yes` or `-y`:

```suggestion
			e.RunCommand("pulumi", "new", template, "-y", "-f", "-s", stackName)
```